### PR TITLE
Using HTTPS for all links to assets.holoviews.org

### PR DESCRIPTION
**Pull request by jlstevens**
PR to address #2464 that greps for ``http://assets.holoviews.org`` and replaces it with ``https://assets.holoviews.org``.

@philippjfr Look right to you? This won't be tested until a doc build is executed and uploaded...

### CODE REVIEW

File: doc/Tutorials/Bokeh_Backend.ipynb [hunk 1]
```diff
@@ -438,7 +438,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "macro_df = pd.read_csv('http://assets.holoviews.org/macro.csv', '\\t')"
+    "macro_df = pd.read_csv('https://assets.holoviews.org/macro.csv', '\\t')"
```
Comment (philippjfr) [inline doc/Tutorials/Bokeh_Backend.ipynb]: Assume these will work fine but worth checking, don't think this is technically necessary.

**Comment by jlstevens** [comment]
Ready to review/merge. Probably worth doing a doc build to check it actually works...

File: examples/gallery/demos/bokeh/mandelbrot_section.ipynb [hunk 2]
```diff
@@ -39,7 +39,7 @@
     "try:    from urllib2 import urlopen\n",
     "except: from urllib.request import urlopen\n",
     "\n",
-    "raw = urlopen('http://assets.holoviews.org/data/mandelbrot.npy').read()\n",
+    "raw = urlopen('https://assets.holoviews.org/data/mandelbrot.npy').read()\n",
```
Comment (philippjfr) [inline examples/gallery/demos/bokeh/mandelbrot_section.ipynb]: Did you test these? There's a few of them.

Comment (jlstevens) [reply examples/gallery/demos/bokeh/mandelbrot_section.ipynb]: No I haven't...I'll just switch those back to http as well...

**Comment by jlstevens** [comment]
I no longer see any code affected other than ``conf.py`` and the submodule reference to builder.

**Comment by philippjfr** [comment]
Great, looks good. Will wait on tests even if it won't make a difference. I've already made a bunch of these changes manually on the website without issue, so it should be fine.

**Comment by philippjfr** [comment]
Merging, will need to check links during a website build before releasing.

**Pull request merged**